### ISSUE TITLE
PNG preview no longer displays the image correctly

### ISSUE THREAD

**Issue by mfigueirido** [vchelaru/FlatRedBall#1411]
It seems like bigger textures show a little bit, but smaller textures can't be seen. It looks like a zooming issue.

![imaxe](https://github.com/vchelaru/FlatRedBall/assets/24922404/d6c69a95-4d22-4e7e-b178-9efa25d8a42a)
![imaxe](https://github.com/vchelaru/FlatRedBall/assets/24922404/161ae0f6-6254-49ca-a88b-161e90c83782)

**Comment by vchelaru** [comment]
This is fixed on the SpeedImprovements branch.

**Comment by mfigueirido** [comment]
Thank you.

I'm on SpeedImprovements but it's not working for me.

![imaxe](https://github.com/vchelaru/FlatRedBall/assets/24922404/a7267206-c806-4f43-bad4-8732acdceac5)

**Comment by vchelaru** [comment]
Oops this was something I fixed locally and forgot to push. It's a Gum fix:

https://github.com/vchelaru/Gum/commit/722ebf8fbed1bafb04962178680974b8952e0b2a

After getting this Gum commit, make sore to rebuild entire solution for FRB. 
Note that this has been merged into FRB NetStandard too, so you don't have to use the SpeedImprovements branch.

![image](https://github.com/vchelaru/FlatRedBall/assets/729631/76c9749c-5d22-4296-8c58-d597f4f36ae3)
